### ISSUE TITLE
Freshness

### ISSUE THREAD

**Issue by TomHaenen** [google/WebFundamentals#6741]
**Page Affected:** https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching

**What needs to be done?**
I was trying to find the answer to a discussion I had with a colleague about freshness. Specifying "max-age" specifies a maximum life time of the cache, which sounds worse than no cache headers. The Mozilla version of this page says that without "max-age" and "expires" it will take (download time - last change date) /10. This page only has "Determine the optimal cache lifetime for each resource".

tl;dr: Which is best: cache header or no cache header.

P.S.: Grammar error in template that was given in this textbox. It says "what's" in: "Please provide description of what's needs to be done here".
